src/vector_index.c

```diff
@@ -71,15 +71,18 @@ IndexIterator *NewVectorIterator(RedisSearchCtx *ctx, VectorFilter *vf) {
   unsigned char *vector = vf->vector;
   switch (vf->type) {
     case VECTOR_TOPK:
+      if (vf->isBase64) {
+        vector = base64_decode(vf->vector, vf->vecLen, &outLen);
+        // TODO: check outLen == expected len.
+        unescape((char *)vector, strlen((char *)vector));
```

Comment:
why are we unescaping here?

Reply:
We have to unescape for `+` and `/` which are part of the `BASE64` standard.

Reply:
Isn't it is only when initially escaping + and / in URL context? Like, when taking a binary buffer (which is not even text), does this have meaning at all?

Reply:
Any string with these characters going through the parser will get an error message if not escape.